csvqb/csvqb/models/cube/__init__.py

```diff
@@ -2,4 +2,4 @@
 from .cube import Cube
 from .catalog import CatalogMetadataBase
 from .csvqb import *
-from .validationerrors import *
+from .validationerrors import DuplicateColumnTitleError, ColumnNotFoundInDataError, MissingColumnDefinitionError
```

Comment:
What motivated you to do this one?

Reply:
I had a circular import problem at one point so I started nerfing star imports until I found the problem. I didn't backtrack once I resolved it.